types/authorization.yaml

```diff
@@ -34,6 +34,7 @@ types:
       - Rejected
       # Occurs when the token has been authorized but no connector is selected within connection_timeout seconds
       - TimedOut
+      - Used
```

Comment:
@hikinggrass @Pietfried should we be here even more explicit and say UsedToStart and UsedToStop?

Reply:
I see, would be indeed better if we keep it and report e.g.

`Processing` > `Accepted` > `UsedToStartTransaction`,
`Processing` > `Accepted` > `TimeOut`